app/Http/routes.php

```diff
@@ -40,10 +40,10 @@
     ]);
 
     // Waiting rooms routes.
+    Route::model('waitingrooms', 'Gladiator\Models\WaitingRoom');
```

Comment:
The `Route::model()` shouldn't matter (I don't think). The custom routes def needed to be before the `Route::resource('waitingrooms'..)`, but I think that's it.